### PR TITLE
[BEAM-14040] [Website] create new page, add link

### PR DESCRIPTION
**Pull request by bullet03**
[BEAM-14040]
- add new page I/O Connectors

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by bullet03** [comment]
R: @AydarZaynutdinov

File: website/www/site/content/en/documentation/io/connectors.md [hunk 1]
```diff
@@ -0,0 +1,921 @@
...
+    <td class="present">✔</td>
+    <td class="absent">✘</td>
+    <td class="present">
+      ✔
+      <a href="https://beam.apache.org/releases/javadoc/2.36.0/org/apache/beam/sdk/io/tika/TikaIO.html">native</a>
+    </td>
+    <td>via X-language</td>
+    <td>via X-language</td>
+    <td class="present">✔</td>
+    <td class="absent">✘</td>
```
Comment (AydarZaynutdinov) [inline website/www/site/content/en/documentation/io/connectors.md:114]:
```suggestion
    <td class="present">✔</td>
```

File: website/www/site/content/en/documentation/io/connectors.md [hunk 2]
```diff
@@ -0,0 +1,921 @@
...
+      <a href="https://beam.apache.org/releases/javadoc/2.36.0/org/apache/beam/sdk/io/parquet/ParquetIO.html">native</a>
+    </td>
+    <td>via X-language</td>
+    <td class="present">✔</td>
+    <td class="absent">✘</td>
+  </tr>
+  <tr>
+    <td>ThriftIO</td>
+    <td class="present">✔</td>
+    <td class="absent">✘</td>
```
Comment (AydarZaynutdinov) [inline website/www/site/content/en/documentation/io/connectors.md:135]:
```suggestion
    <td class="present">✔</td>
```

File: website/www/site/content/en/documentation/io/connectors.md [hunk 3]
```diff
@@ -0,0 +1,921 @@
...
+    </td>
+    <td>via X-language</td>
+    <td>via X-language</td>
+    <td class="present">✔</td>
+    <td class="absent">✘</td>
+  </tr>
+  <tr>
+    <td>S3IO</td>
+    <td class="present">✔</td>
+    <td class="present">✘</td>
```
Comment (AydarZaynutdinov) [inline website/www/site/content/en/documentation/io/connectors.md:148]:
```suggestion
    <td class="present">✔</td>
```

File: website/www/site/content/en/documentation/io/connectors.md [hunk 4]
```diff
@@ -0,0 +1,921 @@
...
+    <td class="present">
+      ✔
+      <a href="https://beam.apache.org/releases/pydoc/2.36.0/apache_beam.io.aws.s3io.html">native</a>
+    </td>
+    <td>via X-language</td>
+    <td class="present">✔</td>
+    <td class="absent">✘</td>
+  </tr>
+  <tr>
+    <td>S3IO</td>
```
Comment (AydarZaynutdinov) [inline website/www/site/content/en/documentation/io/connectors.md:159]:
```suggestion
    <td>GcsIO</td>
```

File: website/www/site/content/en/documentation/io/connectors.md [hunk 5]
```diff
@@ -0,0 +1,921 @@
...
+    <td class="present">✔</td>
+    <td class="present">✘</td>
```
Comment (AydarZaynutdinov) [inline website/www/site/content/en/documentation/io/connectors.md:161]:
```suggestion
    <td class="present">✔</td>
```

File: website/www/site/content/en/documentation/io/connectors.md [hunk 6]
```diff
@@ -0,0 +1,921 @@
...
+      <a href="https://beam.apache.org/releases/pydoc/2.36.0/apache_beam.io.gcp.gcsio.html">native</a>
+    </td>
+    <td>via X-language</td>
+    <td class="present">✔</td>
+    <td class="absent">✘</td>
+  </tr>
+  <tr>
+    <td>HadoopFileSystem</td>
+    <td class="present">✔</td>
+    <td class="present">✘</td>
```
Comment (AydarZaynutdinov) [inline website/www/site/content/en/documentation/io/connectors.md:174]:
```suggestion
    <td class="present">✔</td>
```

File: website/www/site/content/en/documentation/io/connectors.md [hunk 7]
```diff
@@ -0,0 +1,921 @@
...
+      <a href="https://beam.apache.org/releases/pydoc/2.36.0/apache_beam.io.hadoopfilesystem.html">native</a>
+    </td>
+    <td>via X-language</td>
+    <td class="present">✔</td>
+    <td class="absent">✘</td>
+  </tr>
+  <tr>
+    <td>GcsFileSystem</td>
+    <td class="present">✔</td>
+    <td class="present">✘</td>
```
Comment (AydarZaynutdinov) [inline website/www/site/content/en/documentation/io/connectors.md:190]:
```suggestion
    <td class="present">✔</td>
```

File: website/www/site/content/en/documentation/io/connectors.md [hunk 8]
```diff
@@ -0,0 +1,921 @@
...
+      ✔
+      <a href="https://pkg.go.dev/github.com/apache/beam/sdks/v2/go/pkg/beam/io/filesystem/gcs">native</a>
+    </td>
+    <td class="present">✔</td>
+    <td class="absent">✘</td>
+  </tr>
+  <tr>
+    <td>LocalFileSystem</td>
+    <td class="present">✔</td>
+    <td class="present">✘</td>
```
Comment (AydarZaynutdinov) [inline website/www/site/content/en/documentation/io/connectors.md:209]:
```suggestion
    <td class="present">✔</td>
```

File: website/www/site/content/en/documentation/io/connectors.md [hunk 9]
```diff
@@ -0,0 +1,921 @@
...
+      <a href="https://beam.apache.org/releases/javadoc/2.36.0/org/apache/beam/sdk/io/cassandra/CassandraIO.html">native</a>
+    </td>
+    <td>via X-language</td>
+    <td>via X-language</td>
+    <td class="present">✔</td>
+    <td class="absent">✘</td>
+  </tr>
+  <tr>
+    <td>HadoopFormatIO</td>
+    <td class="absent">✘</td>
```
Comment (AydarZaynutdinov) [inline website/www/site/content/en/documentation/io/connectors.md:389]:
```suggestion
    <td class="present">✔</td>
```

File: website/www/site/content/en/documentation/io/connectors.md [hunk 10]
```diff
@@ -0,0 +1,921 @@
...
+    <td class="present">✔</td>
+    <td class="absent">✘</td>
+    <td class="present">
+      ✔
+      native
+    </td>
+    <td>via X-language</td>
+    <td>via X-language</td>
+    <td class="present">✔</td>
+    <td class="absent">✘</td>
```
Comment (AydarZaynutdinov) [inline website/www/site/content/en/documentation/io/connectors.md:828]:
```suggestion
    <td class="present">✔</td>
```

File: website/www/site/content/en/documentation/io/connectors.md [hunk 11]
```diff
@@ -0,0 +1,921 @@
...
+    </td>
+    <td class="present">✔</td>
+    <td class="present">✔</td>
+    <td class="absent">✘</td>
+    <td class="absent">✘</td>
+    <td class="present">
+      ✔
+      native
+    </td>
+    <td class="absent">✘</td>
```
Comment (AydarZaynutdinov) [inline website/www/site/content/en/documentation/io/connectors.md:887]:
```suggestion
    <td class="present">✔</td>
```

**Review by AydarZaynutdinov — CHANGES_REQUESTED** [review]

File: website/www/site/content/en/documentation/io/connectors.md [hunk 12]
```diff
@@ -0,0 +1,42 @@
+# I/O Connectors
```
[the remark this answers is not in the document]
Comment (bullet03) [reply website/www/site/content/en/documentation/io/connectors.md:18]: done

File: website/www/site/content/en/documentation/io/connectors.md (hunk 1 above)
Comment (bullet03) [reply website/www/site/content/en/documentation/io/connectors.md]: done

File: website/www/site/content/en/documentation/io/connectors.md (hunk 2 above)
Comment (bullet03) [reply website/www/site/content/en/documentation/io/connectors.md]: done

File: website/www/site/content/en/documentation/io/connectors.md (hunk 3 above)
Comment (bullet03) [reply website/www/site/content/en/documentation/io/connectors.md]: done

File: website/www/site/content/en/documentation/io/connectors.md (hunk 4 above)
Comment (bullet03) [reply website/www/site/content/en/documentation/io/connectors.md]: done

File: website/www/site/content/en/documentation/io/connectors.md (hunk 7 above)
Comment (bullet03) [reply website/www/site/content/en/documentation/io/connectors.md]: done

File: website/www/site/content/en/documentation/io/connectors.md (hunk 8 above)
Comment (bullet03) [reply website/www/site/content/en/documentation/io/connectors.md]: done

File: website/www/site/content/en/documentation/io/connectors.md (hunk 9 above)
Comment (bullet03) [reply website/www/site/content/en/documentation/io/connectors.md]: done

File: website/www/site/content/en/documentation/io/connectors.md (hunk 10 above)
Comment (bullet03) [reply website/www/site/content/en/documentation/io/connectors.md]: done

File: website/www/site/content/en/documentation/io/connectors.md (hunk 11 above)
Comment (bullet03) [reply website/www/site/content/en/documentation/io/connectors.md]: done

File: website/www/site/content/en/documentation/io/connectors.md (hunk 5 above)
Comment (bullet03) [reply website/www/site/content/en/documentation/io/connectors.md]: done

File: website/www/site/content/en/documentation/io/connectors.md (hunk 6 above)
Comment (bullet03) [reply website/www/site/content/en/documentation/io/connectors.md]: done

**Review by AydarZaynutdinov — APPROVED** [review]
LGTM

File: website/www/site/content/en/documentation/io/connectors.md [hunk 13]
```diff
@@ -0,0 +1,940 @@
...
+    <th>Batch Supported</th>
+    <th>Streaming Supported</th>
+  </tr>
+  <tr>
+    <td>FileIO</td>
+    <td class="present">✔</td>
+    <td class="present">✔</td>
+    <td class="present">
+      ✔
+      <a href="https://beam.apache.org/releases/javadoc/2.36.0/org/apache/beam/sdk/io/FileIO.html">native</a>
```
Comment (AydarZaynutdinov) [inline website/www/site/content/en/documentation/io/connectors.md:43]: Could you please change the collor for the all `native` to the green color?
image

**Review by AydarZaynutdinov — CHANGES_REQUESTED** [review]

File: website/www/site/content/en/documentation/io/connectors.md (hunk 13 above)
Comment (bullet03) [reply website/www/site/content/en/documentation/io/connectors.md:43]: done

**Comment by dariabezkorovaina** [comment]
@pcoet

File: website/www/site/content/en/documentation/io/connectors.md [hunk 14]
```diff
@@ -0,0 +1,940 @@
...
+    <td class="present">✔</td>
+    <td class="present">✔</td>
+  </tr>
+  <tr>
+    <td>KuduIO</td>
+    <td class="present">✔</td>
+    <td class="present">✔</td>
+    <td class="present">
+      ✔
+      <a href=https://beam.apache.org/releases/javadoc/2.36.0/org/apache/beam/sdk/io/kudu/KuduIO.html">native</a>
```
Comment (pcoet) [inline website/www/site/content/en/documentation/io/connectors.md:449]: This link is a 404. Can you test all the links, just to make sure there aren't typos?

File: website/www/site/content/en/documentation/io/connectors.md [hunk 15]
```diff
@@ -0,0 +1,940 @@
...
+    <td>FileIO</td>
+    <td class="present">✔</td>
+    <td class="present">✔</td>
+    <td class="present">
+      ✔
+      <a href="https://beam.apache.org/releases/javadoc/2.36.0/org/apache/beam/sdk/io/FileIO.html">native</a>
+    </td>
+    <td class="present">
+      ✔
+      <a href="https://beam.apache.org/releases/pydoc/2.36.0/apache_beam.io.fileio.html">native</a>
```
Comment (pcoet) [inline website/www/site/content/en/documentation/io/connectors.md:47]: All of your URLs have the version hardcoded: `2.36.0`. Is that intentional? There's also a redirect that uses the `current` slug (in both Java and Python).

**Review by pcoet — CHANGES_REQUESTED** [review]

File: website/www/site/content/en/documentation/io/connectors.md (hunk 14 above)
Comment (bullet03) [reply website/www/site/content/en/documentation/io/connectors.md]: done, tested all the links

File: website/www/site/content/en/documentation/io/connectors.md (hunk 15 above)
Comment (bullet03) [reply website/www/site/content/en/documentation/io/connectors.md]: changed to current

**Review by AydarZaynutdinov — APPROVED** [review]
LGTM

**Review by pcoet — APPROVED** [review]
I didn't do any sanity testing of the links in the staged content, so somebody should do that -- make sure they work and point to the correct content. But at a high level, LGTM. Thanks!

**Review by pcoet — APPROVED** [review]

File: website/www/site/content/en/documentation/io/connectors.md (hunk 14 above)
Comment (pcoet) [reply website/www/site/content/en/documentation/io/connectors.md]: Thanks much!

**Pull request merged**